app/src/main/java/com/tr4n/moviedb/presentation/ui/detail/DetailFragment.kt

```diff
@@ -16,13 +18,21 @@ class DetailFragment :
 
     private val args by navArgs<DetailFragmentArgs>()
 
+    private var tabTitle = listOf(
```

Comment:
readonly, khai báo là val 
```suggestion
    private val tabTitle = listOf(
```